Bumped version in archetypes

svenkubiak opened the pull request:
Updated the versions in both archetypes to the latest version.

Commit:
bumped versions in archetypes

Commit:
added missing artifact

buildhive commented:
[Ninja Web Framework » ninja #281](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/281/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

buildhive commented:
[Ninja Web Framework » ninja #282](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/282/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

Commit:
Merge remote-tracking branch 'upstream/develop' into develop

Conflicts:
	ninja-core/src/site/markdown/developer/changelog.md

buildhive commented:
[Ninja Web Framework » ninja #283](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/283/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

buildhive commented:
[Ninja Web Framework » ninja #284](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/284/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

raphaelbauer commented on ninja-servlet-jpa-blog-archetype/src/main/resources/archetype-resources/pom.xml:
```diff
@@ -143,32 +131,23 @@
         <dependency>
             <groupId>org.webjars</groupId>
             <artifactId>bootstrap</artifactId>
-            <version>3.0.0</version>
```
not sure - but don't  we have to (unfortunately) also replace 3.0.0 and tinyMce's 4.0.16 at defaultLayout.ftl.html ? Did you run the integration tests or the archetype locally?

Commit:
updated templates

svenkubiak commented:
@raphaelbauer 
You are right. I updated the template and also added the jquery webjar which was missing in the jpa archetpye. Integration-Test was successful.

buildhive commented:
[Ninja Web Framework » ninja #286](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/286/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

buildhive commented:
[Ninja Web Framework » ninja #287](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/287/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

raphaelbauer commented:
perfect :) Thanks!
